src/mimefactory.rs

```diff
@@ -42,6 +42,7 @@ pub enum Loaded {
 pub struct MimeFactory<'a> {
     from_addr: String,
     from_displayname: String,
+    sender_displayname: Option<String>, // set only when different to `from_displayname`
```

Comment:
A proper doccomment is better, there is an option to build cargo documentation with all the private structures and fields.

Reply:
i commented in more detail now.